recipes/scriptconfig/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "scriptconfig" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scriptconfig-{{ version }}.tar.gz
+  sha256: d641365029d784cee577cfb8c4a213b83f05e9c2ea17329af0f707941326c975
+
+build:
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - setuptools >=41.0.1
+    - pip
+  run:
+    - python
+    - ubelt >=1.3.6
+    - pyyaml >=5.4.1  # [py<310 and py>=38]
+    - pyyaml >=6.0.0  # [py<312 and py>=310]
+    - pyyaml >=6.0.1  # [py>=312]
```

Comment:
See my comment in https://github.com/conda-forge/staged-recipes/pull/27277/files#r1741559350.
We can run the whole test suit to be sure but I doubt these different versions are really needed.